src/ontology/mondo-edit.obo

```diff
@@ -138261,38 +138261,12 @@ property_value: IAO:0000233 "https://github.com/monarch-initiative/mondo/issues/
-comment: This will be obsoleted in the 2023-10-01 release. The term will be split and the new ID for this term will be MONDO:0800444 (Birt-Hogg-Dube syndrome), and MONDO:0800445 (Birt-Hogg-Dube syndrome 1).
-subset: obsoletion_candidate
-subset: ordo_malformation_syndrome {source="Orphanet:122"}
-synonym: "BHD" RELATED ABBREVIATION [MONDO:Lexical, OMIM:135150]
-synonym: "BHD syndrome" RELATED [GARD:0002322]
-synonym: "Birt Hogg Dube syndrome" RELATED [GARD:0002322]
-synonym: "Birt-Hogg-Dube syndrome" EXACT CLINGEN_PREFERRED [MONDO:Lexical, NCIT:C28244, OMIM:135150]
-synonym: "Birt-Hogg-Dubé syndrome" RELATED [Orphanet:122]
-synonym: "fibrofolliculomas with trichodiscomas and acrochordons" EXACT [OMIM:135150, Orphanet:122]
-synonym: "Hornstein-Knickenberg syndrome" RELATED [OMIM:135150]
-xref: DOID:0050676 {source="MONDO:equivalentTo"}
-xref: EFO:1001273 {source="MONDO:equivalentTo"}
-xref: ICD9:704.8 {source="MONDO:relatedTo", source="MONDO:i2s"}
-xref: MedDRA:10067736 {source="Orphanet:122/e", source="Orphanet:122"}
-xref: MESH:D058249 {source="Orphanet:122/e", source="MONDO:equivalentTo", source="Orphanet:122"}
-xref: NCIT:C28244 {source="MONDO:equivalentTo", source="MONDO:exact-label-match"}
-xref: OMIM:135150 {source="Orphanet:122/e", source="DOID:0050676", source="MONDO:equivalentTo", source="Orphanet:122"}
-xref: Orphanet:122 {source="OMIM:135150", source="MONDO:equivalentTo"}
-xref: SCTID:110985001 {source="MONDO:equivalentTo"}
-xref: UMLS:C0346010 {source="OMIM:135150", source="Orphanet:122/e", source="NCIT:C28244", source="MONDO:equivalentTo", source="MONDO:ncbi_mim2gene_medline", source="Orphanet:122"}
-is_a: MONDO:0000426 {source="DOID:0050676", source="MONDO:Redundant"} ! autosomal dominant disease
-is_a: MONDO:0002254 {source="https://orcid.org/0000-0001-5208-3432"} ! syndromic disease
-is_a: MONDO:0015356 {source="MESH:D058249", source="MONDO:Redundant", source="NCIT:C28244", source="Orphanet:122/inferred"} ! hereditary neoplastic syndrome
-relationship: excluded_subClassOf MONDO:0019300 {source="Orphanet:122"} ! obsolete rare skin tumor or hamartoma
-relationship: has_characteristic HP:0000006 {source="MONDO:HPOA", source="OMIM:135150", source="Orphanet:122"} ! Autosomal dominant inheritance
-relationship: has_material_basis_in_germline_mutation_in http://identifiers.org/hgnc/27310 {source="MONDO:mim2gene_medgen"} ! FLCN
-property_value: confidence "4.7142857142857135" xsd:double
+name: obsolete Birt-Hogg-Dube syndrome
+property_value: IAO:0000231 IAO:0000229
 property_value: IAO:0000233 "https://github.com/monarch-initiative/mondo/issues/6518" xsd:anyURI
 property_value: IAO:0000233 "https://github.com/monarch-initiative/mondo/issues/6749" xsd:anyURI
-property_value: IAO:0006012 "2023-10-01" xsd:string
+is_obsolete: true
+replaced_by: MONDO:0800444
```

Comment:
@nicolevasilevsky this should not be "replaced_by" should be "considered" with both new terms